src/main/webapp/Renderer.js

```diff
@@ -0,0 +1,22 @@
+/*
+** Class for adding the events stored in the calendar to the html page
+*/
+export default class Renderer {
+    constructor(eventList) {
+        this.eventList = eventList;        
+    }
+
+    render(events) {
+        this.eventList.innerHTML = "";
+
+        for (var index in events.list) {
```

Comment:
To get a clearer idea on array traversing, please consult this stack overflow answer (pay attention to the 3rd point): https://stackoverflow.com/a/3010848

Reply:
Thanks a lot! It was very helpul